### DISCUSSION TITLE
How to fix the page for easy debugging?

### DISCUSSION THREAD

**Discussion by Tongzhao9417** [jspsych/jsPsych#3156] [Q&A] [answered]
Hi! I'm a beginner for jsPsych. I have a question about debugging. For example, I need to add CSS to a page that is on the 5th trial. So every time I save it, the page will refresh and I need to keep clicking until I reach the 5th page. Is there any way for me to stay on this page without moving?

**Comment by Tongzhao9417** [comment, marked as answer]
well, I found a way to easy debugging: use jspsych-builder and modulized the code^

**Reply by bjoluc** [reply]
Now you were quicker than me :tada: Glad you found an approach that works for you.

**Reply by Tongzhao9417** [reply]
Thanks for your reply. You are so kind:)
By the way, I want to recommend a repository which beginner can learn faster. https://github.com/coglabuzh/jspsych-tutorial

**Comment by bjoluc** [comment]
Hi @Tongzhao9417,
for quick tests, I typically comment out the lines that add the trials I'd like to skip (say the first four trials) to my `timeline` array when testing. For more advanced stuff, you can also use jsPsych's [simulation mode](https://www.jspsych.org/7.3/overview/simulation/) to simulate some trials and leave out the ones you'd like to debug.
